bus-mapping/src/precompile.rs

```diff
@@ -0,0 +1,23 @@
+//! precompile helpers
+
+use eth_types::Address;
+use revm_precompile::{Precompile, Precompiles};
+
+/// Check if address is a precompiled or not.
+pub fn is_precompiled(address: &Address) -> bool {
+    Precompiles::latest()
```

Comment:
Perhaps we could use `Precompiles::berlin()` here instead of `latest` so that it's obvious which which set of precompiles we have enabled (and to avoid surprises in the future if there are new precompiles and we update the revm dependency)